include/linux/soundwire/sdw_registers.h

```diff
@@ -317,13 +317,15 @@
 #define SDW_CASC_PORT_REG_OFFSET_INTSTAT3	2
 
 /* v1.2 device - SDCA address mapping */
-#define SDW_SDCA_CTL(fun, ent, ctl, ch) (BIT(30) |	\
-					 (((fun) & 0x7) << 22) |	\
-					 (((ent) & 0x40) << 15) |	\
-					 (((ent) & 0x3f) << 7) |	\
-					 (((ctl) & 0x30) << 15) |	\
-					 (((ctl) & 0x0f) << 3) |	\
-					 (((ch) & 0x38) << 12) |	\
-					 ((ch) & 0x07))
+#define SDW_SDCA_CTL(fun, ent, ctl, ch)		(BIT(30) |			\
+						 (((fun) & 0x7) << 22) |	\
+						 (((ent) & 0x40) << 15) |	\
+						 (((ent) & 0x3f) << 7) |	\
+						 (((ctl) & 0x30) << 15) |	\
+						 (((ctl) & 0x0f) << 3) |	\
+						 (((ch) & 0x38) << 12) |	\
+						 ((ch) & 0x07))
```

Comment:
ouch, who designed that register :-/

Reply:
a camel is a horse designed by committee.